backend/src/main/java/com/google/sps/servlets/SearchMatchServlet.java

```diff
@@ -142,7 +159,12 @@ private void sendMatchResponse(HttpServletResponse response, Match match) throws
     matchExists.put(JSON_FIRST_PARTICIPANT_USERNAME, match.getFirstParticipantUsername());
     matchExists.put(JSON_SECOND_PARTICIPANT_USERNAME, match.getSecondParticipantUsername());
     matchExists.put(JSON_DURATION, match.getDuration());
-
+    try {
+      emailNotifier.sendMatchEmail(
+          match.getFirstParticipantUsername(), userService.getCurrentUser().getEmail());
```

Comment:
same comment here as above in sendExpiredResponse()

Reply:
@kmh287 ?